scripts/deploy_factory_dev.sh

```diff
@@ -0,0 +1,229 @@
+# #### Account & Data management for setup
+# #### --------------------------------------------
+# near call $FACTORY_ACCOUNT_ID delete_contract '{"code_hash":"Cn8QJe3SJk36b9NZky4SQuKtu9L2ktYYTCBdy2qKJ8Bq"}' --accountId $FACTORY_ACCOUNT_ID --gas $GAS_100_TGAS
+# near call $FACTORY_ACCOUNT_ID delete_contract '{"code_hash":"8RMeZ5cXDap6TENxaJKtigRYf3n139iHmTRe8ZUNey6N"}' --accountId $FACTORY_ACCOUNT_ID --gas $GAS_100_TGAS
+# near call $FACTORY_ACCOUNT_ID delete_contract '{"code_hash":"ZGdM2TFdQpcXrxPxvq25514EViyi9xBSboetDiB3Uiq"}' --accountId $FACTORY_ACCOUNT_ID --gas $GAS_100_TGAS
+# near call $FACTORY_ACCOUNT_ID delete_contract '{"code_hash":"6SQymHtmezR3u9zAtLBQdb8RWCXxwxnigqSH2mRTxecB"}' --accountId $FACTORY_ACCOUNT_ID --gas $GAS_100_TGAS
+# near delete $FACTORY_ACCOUNT_ID $NEAR_ACCT
+# near create-account $FACTORY_ACCOUNT_ID --masterAccount $NEAR_ACCT --initialBalance 40
+# #### --------------------------------------------
+
+
+
+# #### --------------------------------------------
+# #### Grab the factory v2 code data
+# #### --------------------------------------------
+# http --json post https://rpc.testnet.near.org jsonrpc=2.0 id=dontcare method=query \
+# params:='{"request_type":"view_code","finality":"final","account_id":"'sputnikv2.$FACTORY'"}' \
+# | jq -r .result.code_base64 \
+# | base64 --decode > sputnikdao_factory2_original.wasm
+
+# # Deploy the previous version to allow accurate testing
+# near deploy --wasmFile sputnikdao_factory2_original.wasm --accountId $FACTORY_ACCOUNT_ID --initFunction new --initArgs '{}' --initGas $MAX_GAS
+# #### --------------------------------------------
+
+
+
+# #### --------------------------------------------
+# #### Deploy a v2 DAO & Some proposals
+# #### --------------------------------------------
+# COUNCIL='["'$NEAR_ACCT'"]'
+# TIMESTAMP=$(date +"%s")
+# DAO_NAME=sputnikdao-dev-v2-$TIMESTAMP
+# DAO_ARGS=`echo '{"config": {"name": "'$DAO_NAME'", "purpose": "Sputnik Dev v2 DAO '$TIMESTAMP'", "metadata":""}, "policy": '$COUNCIL'}' | base64`
+# near call $FACTORY_ACCOUNT_ID create "{\"name\": \"$DAO_NAME\", \"args\": \"$DAO_ARGS\"}" --accountId $FACTORY_ACCOUNT_ID --gas $GAS_150_TGAS --amount 10
+# DEMO_DAO_ACCOUNT=$DAO_NAME.$FACTORY_ACCOUNT_ID
+
+# # some sample payouts
+# near call $DEMO_DAO_ACCOUNT add_proposal '{"proposal": { "description": "Sample payment", "kind": { "Transfer": { "token_id": "", "receiver_id": "'$NEAR_ACCT'", "amount": "1000000000000000000000000" } } } }' --accountId $NEAR_ACCT --amount 1
+# near call $DEMO_DAO_ACCOUNT add_proposal '{"proposal": { "description": "Sample payment", "kind": { "Transfer": { "token_id": "", "receiver_id": "'$NEAR_ACCT'", "amount": "1000000000000000000000000" } } } }' --accountId $NEAR_ACCT --amount 1
+# near call $DEMO_DAO_ACCOUNT add_proposal '{"proposal": { "description": "Sample payment", "kind": { "Transfer": { "token_id": "", "receiver_id": "'$NEAR_ACCT'", "amount": "1000000000000000000000000" } } } }' --accountId $NEAR_ACCT --amount 1
```

Comment:
again, not sure why we are having the same command multiple times..

Reply:
could setup a loop, just lazy and need 3 proposals